app/helpers/vita_min_form_builder.rb

```diff
@@ -142,7 +142,7 @@ def vita_min_searchbar(method, label_text, label_icon: "", options: {}, classes:
     return <<~HTML.html_safe
       <div class="vita-min-searchbar form-group#{error_state(object, method)}" role="search">
         <div class="vita-min-searchbar__field">
-          <label class="vita-min-searchbar__label sr-only" for="#{object_name}_#{method}">#{label_text}</label>
+          <label class="vita-min-searchbar__label sr-only.scss" for="#{object_name}_#{method}">#{label_text}</label>
```

Comment:
is the `.scss` how this works? somehow i didn't see it until reviewing just now...

Reply:
oh, this should just be sr-only. I think this is a byproduct of using the refactor tool to rename the file that we forgot to add an extension to.